gdal/frmts/netcdf/netcdflayersg.cpp

```diff
@@ -299,3 +361,17 @@ OGRFeature* netCDFLayer::buildSGeometryFeature(size_t featureInd)
     feat -> SetFID(featureInd);
     return feat;
 }
+
+std::shared_ptr<std::string> netCDFDataset::generateLogName()
+{
+    const char * lognumc;
+
+    if(this->logCount == 0)
```

Comment:
you don't need that complication. CPLGenerateTempFilename() has logic to generate a unique name and has its own counter. You can call it with nullptr. So bsically this method could do just: return CPLGetnerateTempFilename(nullptr);

Reply:
alright thanks, I'll change it to that :+1: